src/main/java/uk/gov/hmcts/darts/common/entity/HearingEntity.java

```diff
@@ -58,9 +55,15 @@ public class HearingEntity {
     @Column(name = "judge_hearing_date")
     private String judgeHearingDate;
 
+    @ManyToMany(fetch = FetchType.EAGER, cascade = {CascadeType.PERSIST, CascadeType.MERGE})
+    @JoinTable(name = "hearing_judge_ae",
+        joinColumns = {@JoinColumn(name = HEA_ID)},
+        inverseJoinColumns = {@JoinColumn(name = "jud_id")})
+    private List<JudgeEntity> judges = new ArrayList<>();
+
     @ManyToMany
     @JoinTable(name = "hearing_media_ae",
-        joinColumns = {@JoinColumn(name = "hea_id")},
+        joinColumns = {@JoinColumn(name = HEA_ID)},
         inverseJoinColumns = {@JoinColumn(name = "med_id")})
     private List<MediaEntity> mediaList;
```

Comment:
Probably need to make this similar conventions to the others? `= new ArrayList<>();`